python/cudf/cudf/core/column/string.py

```diff
@@ -3527,10 +3527,19 @@ def findall(
                 "unsupported value for `flags` parameter"
             )
 
-        data, index = libstrings.findall(self._column, pat, flags)
-        return self._return_or_inplace(
-            cudf.core.frame.Frame(data, index), expand=expand
-        )
+        if expand:
+            warnings.warn(
+                "The expand parameter is deprecated and will be removed in a "
```

Comment:
Can we also open an issue to be able to convert a listcolumn series to dataframe in cudf? Just in case, someone comes back to us in the future with such a behavior request?

Reply:
Nothing that I'm aware of, but pandas does allow this via `apply` : https://stackoverflow.com/a/57498593

Okay to defer this for now 👍